### PR TITLE
Update polkadot library version

### PR DESCRIPTION
**Pull request by DmitriyMolch**
https://cerenetwork.atlassian.net/browse/CBI-2289

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <meta http-equiv="Content-Security-Policy" content="default-src 'self'; connect-src ws://localhost:3000 http://localhost:3000 wss://*.cere.network https://*.infura.io https://*.google-analytics.com wss://*.walletconnect.org https://*.walletconnect.com https://*.sentry.io; style-src 'self' 'unsafe-inline'; img-src 'self' data:; script-src 'self' 'unsafe-inline' 'unsafe-eval' https://*.googletagmanager.com">
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self'; connect-src ws://localhost:3000 http://localhost:3000 ws://127.0.0.1:9944 wss://ext-devs-node-1.cluster-1.cere.network:9945 wss://*.cere.network https://*.infura.io https://*.google-analytics.com wss://*.walletconnect.org https://*.walletconnect.com https://*.sentry.io; style-src 'self' 'unsafe-inline'; img-src 'self' data:; script-src 'self' 'unsafe-inline' 'unsafe-eval' https://*.googletagmanager.com">
```
Comment (AndreiNavoichyk) [inline public/index.html]: Let's also add `wss://ext-devs-node-1.cluster-2.cere.network:9945` and `wss://ext-devs-node-1.cluster-3.cere.network:9945`

File: src/Contexts/Adaptors/EVMAdaptors/EVMDestinationAdaptorProvider.tsx [hunk 2]
```diff
@@ -209,11 +209,13 @@ export const EVMDestinationAdaptorProvider = ({
     const fallback = new Fallback(delayMs, pollingIntervalMs, async () => {
       let res;
       try {
-        res = await destinationBridge?.getProposal(
-          srcChainId,
-          parseInt(depositNonce as string),
-          erc20ProposalHash
-        );
+        if(depositNonce) {
```
Comment (AndreiNavoichyk) [inline src/Contexts/Adaptors/EVMAdaptors/EVMDestinationAdaptorProvider.tsx]: Format the code

Comment (AndreiNavoichyk) [inline src/Contexts/Adaptors/EVMAdaptors/EVMDestinationAdaptorProvider.tsx]: How does it relate to `polkadot/api` lib update?

File: src/Contexts/Adaptors/SubstrateApis/bridgeTypes.json [hunk 3]
```diff
@@ -16,6 +16,7 @@
     "id": "TokenId",
     "metadata": "Vec<u8>"
   },
-  "Address": "IndicesLookupSource",
-  "LookupSource": "IndicesLookupSource"
+  "Address": "MultiAddress",
+  "LookupSource": "MultiAddress",
+  "AccountInfo": "AccountInfoWithDualRefCount"
```
Comment (AndreiNavoichyk) [inline src/Contexts/Adaptors/SubstrateApis/bridgeTypes.json:21]: Why do you need the last?

File: src/Contexts/Adaptors/EVMAdaptors/EVMDestinationAdaptorProvider.tsx (hunk 2 above)
Comment (DmitriyMolch) [reply src/Contexts/Adaptors/EVMAdaptors/EVMDestinationAdaptorProvider.tsx]: parseInt(depositNonce) of undefined returns NaN.  Using NaN  return error in the new polkadot.js library.

File: src/Contexts/Adaptors/SubstrateApis/bridgeTypes.json (hunk 3 above)
Comment (DmitriyMolch) [reply src/Contexts/Adaptors/SubstrateApis/bridgeTypes.json:21]: I added `AccountInfoWithDualRefCount` to provide backward compatibility with the old Cere node. Without that field application returns "invalid address info" error in the console for the old node when integrated with a new polkadot.js version.

File: public/index.html (hunk 1 above)
Comment (DmitriyMolch) [reply public/index.html]: fixed

File: src/Contexts/Adaptors/EVMAdaptors/EVMDestinationAdaptorProvider.tsx (hunk 2 above)
Comment (DmitriyMolch) [reply src/Contexts/Adaptors/EVMAdaptors/EVMDestinationAdaptorProvider.tsx]: Fixed

File: package.json [hunk 4]
```diff
@@ -11,8 +11,8 @@
     "@material-ui/core": "4.12.4",
     "@material-ui/icons": "4.11.3",
     "@material-ui/styles": "4.10.0",
-    "@polkadot/api": "4.2.1",
-    "@polkadot/extension-dapp": "0.37.1",
+    "@polkadot/api": "7.15.2-15",
```
Comment (AndreiNavoichyk) [inline package.json]: Please, check with @MRamanenkau the needed version

File: src/Contexts/Adaptors/SubstrateHomeAdaptor.tsx [hunk 5]
```diff
@@ -280,14 +280,14 @@ export const SubstrateHomeAdaptorProvider = ({
...
-                      const depositNonce = `${response.toJSON()}`;
+                      const depositNonce = response.toString() || undefined
                       setDepositNonce(depositNonce);
                       setHomeTransferTxHash(status.asFinalized.toHex());
                       setTransactionStatus("Transfer to Destination");
                       analytics.trackTransferToDestinationEvent({
                         address,
                         recipient,
-                        nonce: parseInt(depositNonce),
+                        nonce: depositNonce? parseInt(depositNonce) : undefined,
```
Comment (AndreiNavoichyk) [inline src/Contexts/Adaptors/SubstrateHomeAdaptor.tsx]: Please, format the code.

File: package.json (hunk 4 above)
Comment (DmitriyMolch) [reply package.json]: fixed

File: src/Contexts/Adaptors/SubstrateHomeAdaptor.tsx (hunk 5 above)
Comment (DmitriyMolch) [reply src/Contexts/Adaptors/SubstrateHomeAdaptor.tsx]: fixed

**Review by AndreiNavoichyk — COMMENTED** [review]
Update README

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,10 @@
 ## vNext
 ...
 
+## v0.20.0
+
+- Update Cere node version
```
Comment (AndreiNavoichyk) [inline CHANGELOG.md]: Added support of the Cere node v2.32.0

Comment (DmitriyMolch) [reply CHANGELOG.md]: fixed

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**